src/axios-auth.ts

```diff
@@ -11,6 +11,7 @@ instance.interceptors.request.use(
 
     const kcToken = sessionStorage.getItem(SessionStorageKeys.KeyCloakToken)
     request.headers.common['Authorization'] = `Bearer ${kcToken}`
+    request.headers.common['App-Name'] = import.meta.env.APP_NAME
```

Comment:
Sample call to Auth API:
![image](https://github.com/user-attachments/assets/d9024aef-e53d-4041-93d1-e6230fc891c7)

Sample call to Legal API:
![image](https://github.com/user-attachments/assets/2ada7d52-b68f-49ac-9328-3d4f782dba7a)